app/controllers/collections_controller.rb

```diff
@@ -144,7 +144,8 @@ def dataset_download
     time_stamp = DateTime.now.strftime('%Q')
     download_generator = DownloadGenerator.new(time_stamp)
     download_generator.make_archive
-    download_generator.generate_dataset_download(params[:id], request.base_url)
+    admin_download = (!params[:admin].blank? && params[:admin] == "admin")
```

Comment:
Is "!params[:admin].blank?" redundant?

Reply:
Yeah, you're probably right. I'll simplify that.

Reply:
Nevermind. I am merging now.